package.json

```diff
@@ -5,6 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "cross-env NODE_ENV=test && mocha src/test --require @babel/register --recursive --exit || true",
+<<<<<<< HEAD
```

Comment:
You have errors on this file

Reply:
i am still looking at why its there because its not here on my local copy

Reply:
done you can check it now